### DISCUSSION TITLE
keep receive instant rejects after trying with all viable documents :(

### DISCUSSION THREAD

**Discussion by miccio-dk** [community/community#62062] [GitHub Education]
### Hi everyone,

Hey everyone, 
I've been trying to apply for student benefits but I keep getting an instant rejection as per screenshot below:
![image](https://github.com/community/community/assets/10885870/3fbff0bf-9723-48f7-bf8c-219dc42823a2)
I have submitted multiple applications in the past few days, each time using a different document (id card, grade transcript, enrollment letter, courses calendar, university intranet screenshot, etc...) and I am running out of options or ideas.
I tried most of the things mentioned in the link provided in the rejection notification, to no avail.
All the documents I provided clearly feature my full name, university name, and relevant dates.

Surprisingly enough, a few of my colleagues got their applications approved right away, using the same proof of academic status.
My profile features my full name, links to SoMe, and a bio. Is there anything else I could/should try?
The most puzzling thing is that my application gets *instantly* denied, even though it reads [here](https://github.com/orgs/community/discussions/17814) that applications are processed within days to weeks.

Thanks in advance for any help :)

PS: since I have been enjoying the github student benefits before in the past (i.e. during my BSc and MSc - and now I started a PhD), perhaps there is a time limit for the benefits?

**Comment by SmashedFrenzy16** [comment]
Hi @miccio-dk,

Hope you are having a great day!

I would suggest that you first contact your institution for details on how to get the correct ID to enroll, and maybe that will solve your issue. If the issue still persists, you should immediately conatct GitHub Support [here](https://support.github.com/contact?source=subtitle&tags=rr-general-technical)

**Reply by miccio-dk** [reply]
Hi SmashedFrenzy16, 
thanks for the prompt message!
Based on the experiences of my university colleagues, the university-issued ID card should be sufficient.

**Reply by SmashedFrenzy16** [reply]
Hi @miccio-dk,

If the issue persists (which it looks like it still is), the best ting to do is [contact GitHub Support](https://support.github.com/contact?source=subtitle&tags=rr-general-technical) and they can fix the issue with your verification problem.
